test/framework/common.go

```diff
@@ -162,3 +153,54 @@ func wildflyClusterViewTest(t *testing.T, f *framework.Framework, ctx *framework
 
 	return WaitUntilClusterIsFormed(f, t, wildflyServer, "clusterbench-0", "clusterbench-1")
 }
+
+// WildFlySmokeRecoveryScaledownTest runs recovery scale down operation
+func WildFlySmokeRecoveryScaledownTest(t *testing.T, applicationTag string) {
```

Comment:
is the test actually performing recovery during scale down or is it just testing scaling down?

Reply:
It's only testing scale down (which means there is run recovery processing in the background). Do you consider changing name to like `WildflyScaleDownTest` as a better idea?